command/deployment_status.go

```diff
@@ -48,6 +48,9 @@ Status Options:
   -monitor
     Enter monitor mode to poll for updates to the deployment status.
 
+  -wait-time
+    How long to wait before polling an update, used in conjunction with monitor mode. Defaults to 2s.
```

Comment:
Two requests here: let's change the name to `-wait` as that matches what we use for similar configuration values elsewhere. And let's wrap this to 80col:

```suggestion
-wait
    How long to wait before polling an update, used in conjunction with monitor 
    mode. Defaults to 2s.
```

(And then we'll need to change the actual flag parsing to match.)

Reply:
Makes sense :blush: 

I debated myself whether to go with `wait`, but I always seem to land on the more verbose options :smile: This has changed now :rocket: